catalyst-uploader.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+	"time"
+
+	"github.com/livepeer/catalyst-uploader/core"
+	"github.com/livepeer/go-tools/drivers"
+)
+
+var Version string
+
+const WaitBetweenWrites = 5 * time.Second
+const WriteTimeout = 30 * time.Second
```

Comment:
Is this enough? I think our upload times in other places are minutes-long right? is it only due to huge >GB files which I hope we won't get for segments?

Reply:
Yeah, this is segments and manifests and so 30s should be more than enough